kickstarts/almalinux-8-micro.aarch64.ks

```diff
@@ -75,22 +64,21 @@ echo '%_install_langs en_US.UTF-8' > /etc/rpm/macros.image-language-conf
 # force each container to have a unique machine-id
 > /etc/machine-id
 
-# create tmp directories because there is no tmpfs support in Docker
-umount /run
-systemd-tmpfiles --create --boot
-
-# disable login prompt and mounts
-systemctl mask console-getty.service \
-               dev-hugepages.mount \
-               getty.target \
-               systemd-logind.service \
-               sys-fs-fuse-connections.mount \
-               systemd-remount-fs.service
-
 # remove unnecessary files
-rm -f /var/lib/dnf/history.* \
+rm -rfv /var/lib/dnf \
```

Comment:
Why do we need verbosity (`-v`) here?

Reply:
To see, what is removed in /root/anaconda-post.log file.